.github/workflows/ci.yaml

```diff
@@ -63,10 +63,13 @@ jobs:
       - name: build and import image
         run: |
           make images
-          docker save everoute/release | sudo k3s ctr images import -
+          docker tag everoute/release everoute/release:latest
+          docker save everoute/release:latest | sudo k3s ctr images import -
 
       - name: make kubernetes deploy test
-        run: make deploy-test
+        run: |
+          sed -i 's/# imagePullPolicy: IfNotPresent/imagePullPolicy: IfNotPresent/g' deploy/everoute.yaml
```

Comment:
why remove `imagePullPolicy: IfNotPresent` in deploy/everoute.yaml and use sed here?

Reply:
ditto. for test convenience, It will be removed.